variables.tf

```diff
@@ -170,3 +170,31 @@ variable "map_roles" {
   }))
   default = []
 }
+
+variable "enable_weave_scope" {
+  description = "Enable or disable Weave scope monitoring"
+  type        = bool
+  default     = false
+}
+
+variable "namespace" {
+  type    = string
+  default = "default"
+}
+
+variable "weave_scope" {
+  description = "Additional IAM roles to add to the aws-auth configmap."
+  type = list(object({
+    enabled          = bool
+    create_namespace = bool
+    namespace        = string
+  }))
+  default = [
+    {
+      enabled          = false
```

Comment:
I don't think we need this here, if used along with cluster setup then all those variables will have specific values which do not need to be modified.